doc/data/messages/p/prefer-typing-namedtuple/good.py

```diff
@@ -0,0 +1,7 @@
+from typing import NamedTuple
+
+
+class Philosophy(NamedTuple):
+    goodness: str
+    truth: bool
+    beauty: float
```

Comment:
A link for related.rst: https://docs.python.org/3/library/collections.html#namedtuple-factory-function-for-tuples-with-named-fields (in the "see also")

Reply:
What about linking to `typing.NamedTuple` instead?

Reply:
Yeah, that make sense. Or both, I'll let you choose :)